ccvm/vm.go

```diff
@@ -190,6 +190,9 @@ func startHTTPServer(ctx context.Context, resultCh chan interface{}, downloadCh
 			_ = listener.Close()
 			return
 		}
+		if strings.HasPrefix(line, "MESSAGE:") {
```

Comment:
Not sure why it crashes but this code is not correct.  The issue is is that this code runs inside a systemd user service.  If you print something to standard output it will appear in the systemd logs and not on the user's terminal.  To send data to the terminal that issue create you need to send the string down the result channel, e.g.,

```
if strings.HasPrefix(line, "MESSAGE:") {
 			resultCh <- types.CreateResult{
 				Line: fmt.Sprintf("[%s]\n", line[len("MESSAGE:"):]),
 			}
} else if line == "OK" || line == "FAIL" {

Reply:
You also need to skip the first few characters of the string that contain the text "MESSAGE:"